_edgedb_js/index.js

```diff
@@ -0,0 +1,197 @@
+"use strict";
+
+const connect = require("edgedb").default;
+
+class Connection {
+  constructor(opts) {
+    this.connection = connect({
+      host: "localhost",
+      port: 5656,
+      username: "edgedb",
+      database: "edgedb_bench",
+      ...(opts || {})
+    });
+  }
+
+  async connect() {
+    if (this.connection instanceof Promise) {
+      this.connection = await this.connection;
+    }
+  }
+
+  async userDetails(id) {
+    return await this.connection.fetchOne(
```

Comment:
Wait... This should either be `fetchOneJSON` (one kind of benchmark), or you should call JSON.stringify() on the result of `fetchOne()`

Reply:
I've updated the code so that the output of the "apps" is a string of JSON. There are 2 versions of EdgeDB benchmark one fetching JSON directly another stringifying the Objects.

Postgres benchmark also comes in 2 flavors now - one that mimics the Postgres psycopg benchmark in Python (making multiple queries without transactions) and the other is transactional.

Reply:
Just make all raw sql benchmarks transactional. And then merge!

Reply:
OK, dropped the non-transactional Postgres benchmark.